### PR TITLE
complete the README.md

### PR DESCRIPTION
**Pull request by cthiriet**
#3

### CODE REVIEW

**Comment by Tcyla** [comment]
Cool!

**Review by prudhomm — COMMENTED** [review]
this is good

File: README.md [hunk 1]
```diff
@@ -13,100 +16,104 @@ GitHub is a way to use the same power of Git all online with an easy-to-use inte
...
 
-Repositories also contain **README**s. You can add a README file to your repository to tell other people why your project is useful, what they can do with your project, and how they can use it. We are using this README to communicate how to learn Git and GitHub with you. 😄 
-To learn more about repositories read ["Creating, Cloning, and Archiving Repositories](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-repositories) and ["About README's"](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes). 
+Repositories also contain **README**s. You can add a README file to your repository to tell other people why your project is useful, what they can do with your project, and how they can use it. We are using this README to communicate how to learn Git and GitHub with you. 😄
+To learn more about repositories read ["Creating, Cloning, and Archiving Repositories](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-repositories) and ["About README's"](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes).
```
Comment (prudhomm) [inline README.md]: why ?

Comment (prudhomm) [reply README.md]: @ClemDev2000 ok for u ?

**Review by prudhomm — CHANGES_REQUESTED** [review]

**Pull request closed without merging**